docs/courses/README.md

```diff
@@ -1,28 +1,104 @@
+
+These concepts represent a hierarchical structure to LibreLingo course material: `Courses > Modules > Skills`
+
+- The **target language** is the language the course is meant to teach.
+- The **source langauge** is a language that users of the course are assumed to know.
+
+The course directory name is usually structured in `<destination language>-from-<target audience's language>`. e.g. `french-from-english` teaches French to English speakers.
+
+<a id="exploring-courses"></a>
+### Exploring the courses
+
+To have a better understanding of how courses work, you can explore how the course files are organized on this repository.
+
+On the `courses` directory of [this repository](https://github.com/kantord/LibreLingo/tree/main/courses) you will find courses. Under each course you'll find modules. And inside each module you'll find skills, which are saved as individual .yaml files.
+
+Now let's look into how they look on the frontend, to the user.
+
+If you go to the [homepage](https://librelingo.app/), courses are listed there. e.g. You can see a "START LEARNING SPANISH" button. (**NOTE:** Depending on the state and completeness of the courses, all of them might not be shown there on the website.) So basically, there are courses for each language. You can think of "courses" as "languages" on your mind if that's more comfortable to you.
+
+When you enter a course page, for example the Spanish course [here](https://librelingo.app/course/spanish-from-english/), you'll see headings like Basics, Introduction, Activities. These are modules. And under each module heading there are a list of skills under it. Modules are merely there to serve the purpose of organizing skills and basically nothing else. Think of them as folders for your skills.
+
+"Skills" are the actual lessons focused on a particular topic. Each skill is like a set of questions that follow a particular theme. For example, "Animal" skill has questions relating to animals, "Food" skill has questions relating to foods. Each skill is designed to test the skill on a certain topic.
+
+Inside a skill YAML file there are some keys that you might want to learn of, which is exaplained in detail [here](course.md).
+
+<a id="things-new-contributors"></a>
+### Things you can do as a new contributor
+
+If you are new in contributing to course data, these are the things you can do to get a proper understanding of the courses and have a good time with the project:
+
+- Go to [LibreLingo's Spanish course](https://librelingo.app/course/spanish-from-english) and try out some of the skills. Just get an idea on how the questions are structured and the type of answers made available.
+- Clone this repository or [download it](https://github.com/kantord/LibreLingo/archive/main.zip) and study the `courses` directory and how everything's laid out. Be sure to have a good text editor, such as [Notepad++](https://notepad-plus-plus.org/), [Geany](https://geany.org/) in hand to view or edit files.
```

Comment:
Do we really want to encourage people to clone the repository? I think it's not really necessary, because on GitHub, you can simply edit files on the GUI and submit merge requests from there :thinking: 

If you are creating a new course from scratch, it's probably good to know how to clone the repo, but just to add contributions to an existing one, it might not be necessary, no?

Reply:
I did saw something like that. Some questions:

1. Does it support editing multiple files and include it in one commit? Can the commit be done on multiple sessions? (usually translating is a long process and it may take multiple days/sessions)
2. Can the commit be updated later (amend)?

Reply:
> Does it support editing multiple files and include it in one commit?

No, probably not. [Unless you use Codespaces I guess](https://github.com/features/codespaces).

> Can the commit be done on multiple sessions? (usually translating is a long process and it may take multiple days/sessions)

Yes, I think you can create a draft PR and keep editing the same file in the PR. Actually that way you can also edit multiple files I think

> Can the commit be updated later (amend)?

I don't think so, however you can have multiple commits in an MR I think, and you can squash them when merging

Reply:
I had some thought.

> Do we really want to encourage people to clone the repository?

Since this is about just exploring, we can skip cloning anyway. We can use a link to browse:

```suggestion
- [Browse this repository](https://github.com/kantord/LibreLingo/) or [download it](https://github.com/kantord/LibreLingo/archive/main.zip) and study the `courses` directory and how everything's laid out. Be sure to have a good text editor, such as [Notepad++](https://notepad-plus-plus.org/), [Geany](https://geany.org/) in hand to view or edit files.
```

I kept the download as a second option here, since it is faster to browse and people can use their favorite text editor to see file contents in an environment they already use everyday. But let me know what you think.

We also have some cloning instructions in `editing-courses.md`. I personally don't use the GitHub GUI options so I'm not aware of its features. If you think it's more intuitive or easier to use, then we might have to make changes on that file for GitHub GUI. The instructions are meant for the beginners (advanced users probably won't need it), so maybe that could be better. Or we can keep existing cli commands in a separate title. I think you know your user-base better than I do, so I leave the decision up to you.

Reply:
I've made the change in my last suggested change. Please let me know if this should be discussed more.

And I think we've had a rough discussion about including Git cli instructions on a separate file (on a new MR). So we can probably include GitHub GUI options here on a separate MR too. Please resolve this if this is ok for now.